jerry-core/debugger/jerry-debugger-ws.c

```diff
@@ -13,8 +13,6 @@
  * limitations under the License.
  */
 
-#ifdef JERRY_DEBUGGER
```

Comment:
I see no problem with the non-cmake use case. On the contrary, then those who don't use cmake, don't have to keep in mind which files they may add to the list of sources and which they must not. (Right now, if someone just gathers the list of _all_ files, including the debugging-related ones, and tries to build them without `JERRY_DEBUGGER` defined - that's the non-cmake use case -, then s/he will get an error. With this patch, that error goes away.)